tour-tip/test/get-tip-model.spec.js

```diff
@@ -0,0 +1,34 @@
+/*global describe,afterEach,beforeEach,it,expect*/
+
+import {default as getTipModelDefault, getById, getRandomOfSize} from '../get-tip-model';
+
+describe('tour tip get tip model', () => {
+
+	describe('getById', () => {
+
+		it('gets by ID', () => {
+			expect(getById('topics-m').id).to.equal('topics-m');
+		});
+
+		it('returns undefined if the ID does not exist', () => {
+			expect(getById('???')).to.be.undefined;
+		});
+
+		it('is the default function', () => {
+			expect(getTipModelDefault === getById).to.be.true;
+		});
+
+	});
+
+	describe('getRandomOfSize', () => {
```

Comment:
_Very_ weak testing on this function but marginally better than no testing at all. Wasn't able to successfully mock out the config file, despite banging my head against my keyboard for some time :confounded: